script/fetchneedles

```diff
@@ -57,6 +57,7 @@ do_fetch() {
     [ "$needles_separate" = 1 ] || return 0
     if test -d products; then
         for nd in products/*/needles; do
+            ! [ -L "$target/$nd" ] || continue
```

Comment:
I don't like the double-negative. Besides, this is Bash so for even more performance I'd use `[[`:  
```
[[ -h $target/$nd ]] && continue`
```

Reply:
`-h` and `-L` are synonyms afair.

I second avoiding the double negative here, though.

Reply:
I never used `-L` with the Bash-internal version so to be safe I changed it to `-h`.

Reply:
Just because you never used `-L` is not a very convincing argument for me to change something … but if it makes you happy :) I changed the code to prevent the double negation and used `-h` now. However I have not used `[[` as instead I made the script fully POSIX sh compatible – for performance reasons as you mentioned :).